lib/core/renderer.js

```diff
@@ -413,17 +413,19 @@ export default class Renderer {
         throw e
       }
     }
-    const options = {
+    const options = Object.assign({
       resources: 'usable', // load subresources (https://github.com/tmpvar/jsdom#loading-subresources)
       runScripts: 'dangerously',
       beforeParse(window) {
         // Mock window.scrollTo
         window.scrollTo = () => {}
       }
-    }
+    }, opts)
     const jsdomErrHandler = (err) => { throw err }
-    if (opts.virtualConsole !== false) {
-      options.virtualConsole = new jsdom.VirtualConsole().sendTo(consola)
+    if (options.virtualConsole !== true) {
+      if (options.virtualConsole === true) {
```

Comment:
Dead code?

Reply:
Whoops. Should be `!== false`

Reply:
But then I can omit the explicit comparison. Now it should be fine